frame/staking/src/pallet/mod.rs

```diff
@@ -431,6 +437,27 @@ pub mod pallet {
 		ValueQuery,
 	>;
 
+	/// Paginated exposure of a validator at given era.
+	///
+	/// This is keyed first by the era index to allow bulk deletion, then the stash account and
+	/// finally the page.
+	///
+	/// It is removed after `HISTORY_DEPTH` eras.
+	/// If stakers hasn't been set or has been removed then empty exposure is returned.
+	#[pallet::storage]
+	#[pallet::getter(fn paged_eras_stakers)]
+	#[pallet::unbounded]
+	pub type PagedErasStakers<T: Config> = StorageNMap<
+		_,
+		(
+			NMapKey<Twox64Concat, EraIndex>,
+			NMapKey<Twox64Concat, T::AccountId>,
+			NMapKey<Twox64Concat, PageIndex>,
+		),
+		Exposure<T::AccountId, BalanceOf<T>>,
```

Comment:
I want to use the same exposure instead of holding validator balance separately in another storage item. We should make sure validator balance is only on the first page, and the subsequent pages will have `own=0`. And we should still guarantee for each exposure page, `total = own + sum_of(others)`.

We can improve on it by removing `own` altogether. Side effect is though, validator will receive two payments for each era:
- the commission payout.
- payout for their `own` stake while going through individual exposure.

Reply:
> and the subsequent pages will have own=0

I get that this sounds clever, but bear in mind that we are also storing an extra `own=0` in multiple places.

I think end of the day if you can pull this off, it will be beneficial as the dominant factor will be avoiding the extra IO of the storage read. Decoding a few extra `0u128`s here and there is totally fine.